core/src/main/java/com/novoda/noplayer/exoplayer/InfoForwarder.java

```diff
@@ -0,0 +1,221 @@
+package com.novoda.noplayer.exoplayer;
+
+import android.view.Surface;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.Format;
+import com.google.android.exoplayer2.decoder.DecoderCounters;
+import com.google.android.exoplayer2.upstream.DataSpec;
+import com.novoda.noplayer.listeners.InfoListeners;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+class InfoForwarder extends ExoPlayerEventForwarder {
+
+    private final InfoListeners infoListeners;
+
+    InfoForwarder(InfoListeners infoListeners) {
+        this.infoListeners = infoListeners;
+    }
+
+    @Override
+    public void onLoadStarted(DataSpec dataSpec, int dataType, int trackType, Format trackFormat, int trackSelectionReason,
+                              Object trackSelectionData, long mediaStartTimeMs, long mediaEndTimeMs, long elapsedRealtimeMs) {
+        HashMap<String, String> keyValuePairs = new HashMap<>();
```

Comment:
All of these methods were originally exposed to `NoPlayer` when they should not be. These are specific to `ExoPlayer` and we should be providing a `NoPlayer` version, this is that 😄